socorro/unittest/external/boto/test_connection_context.py

```diff
@@ -8,8 +8,11 @@
 
 from socorrolib.lib.util import DotDict
 from socorro.external.boto.connection_context import (
+    DatePrefixKeyBuilder,
+    KeyBuilderBase,
+    KeyNotFound,
     S3ConnectionContext,
-    RegionalS3ConnectionContext,
+    RegionalS3ConnectionContext
```

Comment:
A trailing comma is a good thing. It means, next time someone adds to that list, their diff is less.